docs/content/doc/installation/from-binary.en-us.md

```diff
@@ -34,12 +38,21 @@ After following the steps above you will have a `gitea` binary within your worki
 
 ### Old glibc versions
 
-Older Linux distributions (such as Debian 7 and CentOS 6) may not be able to load the Gitea binary, usually resulting an error like ```./gitea: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.14' not found (required by ./gitea)```. This is due to the integrated SQLite support in the binaries we provide. In the future, we will provide binaries without the requirement for glibc. As a workaround, you can upgrade your distribution or [install from source]({{< relref "from-source.en-us.md" >}}).
+Older Linux distributions (such as Debian 7 and CentOS 6) may not be able to load the
+Gitea binary, usually producing an error such as `./gitea: /lib/x86_64-linux-gnu/libc.so.6:
+version `GLIBC\_2.14' not found (required by ./gitea)`. This is due to the integrated
+SQLite support in the binaries provided by dl.gitea.io. In this situation, it is usually
+possible to [install from source]({{< relref "from-source.en-us.md" >}}) without sqlite
+support.
```

Comment:
Escaping the \` will not work in this case. Use the `<code>` tag or change the quote before GLIBC to be a `'`